docs/README.md

```diff
@@ -1,14 +1,8 @@
 # KSQL Documentation
 The KSQL documentation is available on the Confluent Platform documentation site at [docs.confluent.io](https://docs.confluent.io/current/ksql/docs/index.html).
 
-## Interoperability
-
-This table shows the version compatibility matrix of which Kafka clusters can be used to read from and write into while running KSQL queries.
-
-|        KSQL        |        0.1       |        0.2       |        0.3       |        0.4       |        0.5       |
-|:------------------:|:----------------:|:----------------:|:----------------:|:----------------:|:----------------:|
-|    Apache Kafka    | 0.10.1 and later | 0.10.1 and later | 0.10.1 and later | 0.10.1 and later |0.10.1 and later |
-| Confluent Platform | 3.1.0 and later  | 3.1.0 and later  | 3.1.0 and later  | 3.1.0 and later  | 3.1.0 and later  |
+# Generating KSQL docs for Confluent Platform
+The KSQL docs are imported from this public repo and built along with the rest Confluent Platform documentation. 
```

Comment:
Typo, should be: "along with the rest [of the] Confluent Platform documentation."